deploy/submariner/lighthouse_operator_cluster_role.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: submariner-operator:lighthouse
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
```

Comment:
can we add a comment explaining that we need this access to update the coredns configuration on the cluster?

Reply:
Done.